public/js/components/Utilities/ShowAllErrors.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import flattenFormErrors from '../../util/flattenFormErrors';
+
+class ShowAllErrors extends React.Component {
```

Comment:
It looks like this is just a glorified list. How about factoring out the error stuff, so that it becomes more reusable? (and you can avoid having to deep-flatten twice).

Reply:
good shout!